docs/reference/query-dsl/script-score-query.asciidoc

```diff
@@ -74,6 +74,108 @@ to be the most efficient by using the internal mechanisms.
 --------------------------------------------------
 // NOTCONSOLE
 
+[[vector-functions]]
+===== Distance functions for vector fields
+These functions are used to calculate distances
```

Comment:
Let's maybe avoid mentioning "distance" since eg. cosineSimilarity measure the similarity between two vectors rather than their distance?